Mathlib/MeasureTheory/Function/LpSpace/DomAct/Basic.lean

```diff
@@ -0,0 +1,120 @@
+/-
+Copyright (c) 2023 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import Mathlib.MeasureTheory.Integral.Bochner
+import Mathlib.MeasureTheory.Function.AEEqFun.DomAct
+
+/-!
+# Action of `Mᵈᵐᵃ` on `Lᵖ` spaces
+
+In this file we define action of `Mᵈᵐᵃ` on `MeasureTheory.Lp E p μ`
+given by `mk c • ⟨⟨f, _⟩, _⟩ = ⟨⟨(f <| c • ·), _⟩, _⟩`.
```

Comment:
I think it is important to make the doc string human-friendly and I claim this line is not. How about:
```suggestion
If `f : α → E` is a function representing an equivalence class in `Lᵖ(α, E)`, `M` acts on `α`, and
`c : Mᵈᵐᵃ`, then `c • [f]` is represented by the function `a ↦ f (c • a)`.

```